### PR TITLE
CU-2a7h6nb, CU-2a221ht, CU-2e0d352

### PR DESCRIPTION
**Pull request by cristhianescobar**
CU-2e0d352 - Start a core ui module with some common composables
CU-2a221ht - Moving login feature to a separate module, create resources and utilities core modules
CU-2a7h6nb - Created a Resources core module to house all the resources, drawables, values etc…

### CODE REVIEW

**Comment by newmskywalker** [comment]
Task linked: [CU-2a221ht Move Login functionality to it's own module](https://app.clickup.com/t/2a221ht)

**Comment by newmskywalker** [comment]
Task linked: [CU-2e0d352 Start a core-ui module](https://app.clickup.com/t/2e0d352)

**Comment by newmskywalker** [comment]
Task linked: [CU-2a7h6nb Start A Core Resources Module](https://app.clickup.com/t/2a7h6nb)

File: android/core-ui-utils/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    kotlin(Plugins.android)
+}
+
+android {
+    compileSdk = Versions.compileSdk
+
+    defaultConfig {
+        minSdk = Versions.minSdk
+        targetSdk = Versions.targetSdk
+        resourcePrefix = "ui-utils"
```
Comment (andymiller813) [inline android/core-ui-utils/build.gradle.kts]: I think this should be "core-ui"

File: android/core-ui-utils/src/main/java/io/projectnewm/core/ui/utils/Brush.kt [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    return Brush.verticalGradient(
+        colors = listOf(
+            colorResource(id = R.color.purple_artist_background_1),
+            colorResource(id = R.color.purple_artist_background_2)
+        )
+    )
+}
+
+@Composable
+fun RingDecorator(
```
Comment (andymiller813) [inline android/core-ui-utils/src/main/java/io/projectnewm/core/ui/utils/Brush.kt]: I think this is unused now

Comment (andymiller813) [inline android/core-ui-utils/src/main/java/io/projectnewm/core/ui/utils/Brush.kt]: I think this is unused now

File: android/core-ui-utils/src/main/java/io/projectnewm/core/ui/utils/Brush.kt [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            colorResource(id = R.color.orange_song_ring_2),
+            colorResource(id = R.color.orange_song_ring_1),
+            colorResource(id = R.color.orange_song_ring_2),
+            colorResource(id = R.color.orange_song_ring_1),
+        )
+    )
+}
+
+@Composable
+fun ArtistRingBrush(): Brush {
```
Comment (andymiller813) [inline android/core-ui-utils/src/main/java/io/projectnewm/core/ui/utils/Brush.kt]: I think this is also unused now

Comment (andymiller813) [inline android/core-ui-utils/src/main/java/io/projectnewm/core/ui/utils/Brush.kt]: I think this is also unused now

**Review by andymiller813 — APPROVED** [review]
Couple minor comments otherwise LGTM

**Review by newmskywalker — APPROVED** [review]
💯

**Pull request merged**